core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -3980,6 +4063,114 @@ paths:
                 $ref: '#/components/schemas/ErrorReport'
         500:
           $ref: '#/components/responses/RawlsInternalError'
+  /api/workspaces/v2/{workspaceNamespace}/{workspaceName}/bucketMigration:
+    get:
+      tags:
+        - workspaces_v2
+      summary: list bucket migration attempts for workspace
```

Comment:
What does this return for a workspace that has never been scheduled for migration?

Reply:
Yes, that's correct